packages/forklift-console-plugin/src/modules/Providers/views/create/components/EsxiProviderCreateForm.tsx

```diff
@@ -101,18 +95,13 @@ export const EsxiProviderCreateForm: React.FC<EsxiProviderCreateFormProps> = ({
       if (id == 'sdkEndpoint') {
         const sdkEndpoint = trimmedValue || undefined;
 
-        // Revalidate URL - VCenter or ESXi
-        const trimmedURL = provider?.spec?.url?.trim() || '';
-        const validationState = validateEsxiURL(trimmedURL);
-
-        dispatch({ type: 'SET_FIELD_VALIDATED', payload: { field: 'url', validationState } });
-
         onChange({
           ...provider,
           spec: {
+            ...provider?.spec,
```

Comment:
see comment above, removing, `type` and `url` makes more sense here as they do nothing (before and after this change).